smallrye-reactive-messaging-amqp/src/test/java/io/smallrye/reactive/messaging/amqp/AmqpBroker.java

```diff
@@ -33,6 +33,7 @@ public Map<String, String> start(int port) {
                 config.setLargeMessagesDirectory("target/data/large-messages");
                 config.addAcceptorConfiguration("in-vm", "vm://0");
                 config.addAcceptorConfiguration("tcp", "tcp://127.0.0.1:" + port);
+                config.setMaxDiskUsage(99);
```

Comment:
That's a sneaky way to postpone the cleanup of your file system :-D

Reply:
:) I blame the openshift local